### DISCUSSION TITLE
Support a libSQL adapter

### DISCUSSION THREAD

**Discussion by TzviPM** [mikro-orm/mikro-orm#5283] [Ideas]
**Is your feature request related to a problem? Please describe.**

I would like to easily use a sqlite database in development, but connect to a different remote database in production. libSQL would be a nice tool to use that adds some consistency, but there is currently no official libSQL support in MikroORM.

**Describe the solution you'd like**

While custom adapters are supported, I fell that a libSQL adapter would be a reasonable addition to the core offering. I am currently looking to build this out [in the knex library](https://github.com/knex/knex/issues/5808), and I would love to contribute a `@mikro-orm/libsql` package here if you'd be open to that contribution.

**Describe alternatives you've considered**

Custom adapter

**Additional context**

No additional context, except to thank you for making such an awesome library :)

**Comment by B4nan** [comment]
How is this different from the two sqlite drivers we already have? If you can make it work with knex, adding the driver should be rather simple, but I'd first want to see an actual reason.

edit: gave it a quick look, and to me it sounds more like you want "turso driver", as that seems to be the only reason to use that sqlite fork, or are there any other alternatives you could use?
edit2: in case you wouldnt get a response from knex maintainers, we could surely have the new dialect added from inside the ORM, it can be provided dynamically, also take a look at how mariadb works, that is a similar case, knex does not have a native driver, so we create it dynamically from the mysql one)

https://github.com/mikro-orm/mikro-orm/blob/master/packages/mariadb/src/MariaDbConnection.ts#L13

**Comment by B4nan** [comment]
Next version will include libsql driver: https://github.com/mikro-orm/mikro-orm/pull/5417
